hack/create.sh

```diff
@@ -14,16 +14,6 @@ if [[ -z "$AZURE_TENANT_ID" ]]; then
     exit 1
 fi
 
-if [[ -z "$AZURE_CLIENT_ID" ]]; then
```

Comment:
Why this diff? Can you just add the +x -x block?

Reply:
for all vars? I was thinking moving all sensitive data under one block just after this one

Reply:
Not all vars, just these two.

Reply:
Also I think we need to gate subscription and tenant id